Section1_lab4_ShanHe_JoannaHuang_TiffanyJaya.Rmd

```diff
@@ -278,7 +280,113 @@ Although the assumption is not met, given the substantial sample size, we can be
 
 Since all six assumptions of the Classical Linear Model are met, we can assume that the OLS estimators are consistent, normally distributed and BLUE.
 
-# Conclusion: 
+# Model 2
+One model that includes key explanatory variables and only covariates that you believe increase the
+accuracy of your results without introducing bias (for example, you should not include outcome variables
+that will absorb some of the causal effect you are interested in). This model should strike a balance
+between accuracy and parsimony and reflect your best understanding of the determinants of crime. 
+
+# Model 3: most, if not all, other covariates
+
+Adding arbitrary variables uses up valuable degrees of freedom which increases the standard error which in turn reduces the precision of this estimate. This effect is especially pronounced in small sample size. 
+
+One model that includes the previous covariates, and most, if not all, other covariates. A key purpose
+of this model is to demonstrate the robustness of your results to model specification.
+
```

Comment:
I recommend to remove this paragraph because it does not jive with the rest of the paper, partly because it is a run-on sentence.